CMakeLists.txt

```diff
@@ -399,7 +397,7 @@ message("HALF_INCLUDE_DIR: ${HALF_INCLUDE_DIR}")
 option( MIOPEN_DEBUG_FIND_DB_CACHING "Use system find-db caching" ON)
 
 set( MIOPEN_INSTALL_DIR miopen)
-set( DATA_INSTALL_DIR ${MIOPEN_INSTALL_DIR}/${CMAKE_INSTALL_DATAROOTDIR}/miopen )
+set( DATA_INSTALL_DIR ${CMAKE_INSTALL_DATAROOTDIR}/${MIOPEN_INSTALL_DIR})
```

Comment:
This should not be removed. The data files should not be put directly in the data directory.